_skycap/the-basics/formations-stencils-and-snapshots.md

```diff
@@ -0,0 +1,121 @@
+
+However, this might not always be what you need. You might want to render and apply only a subset of the Stencils available in a Formation.  For example when deploying a single service or running a database migration job, you may not want to deploy the entire stack.
+
+StencilGroups help you group the Stencils of a Formation into dynamic or static subsets so they can be rendered together.
+
+## What is a Snapshot?
+
+A Snapshot is the state of your application at a point in time. It includes all your applications images and their unique tags, environment variables and configuration items that are defined in your Skycap application. 
+
+<img src="/assets/skycap/retagging.png" width="600px">
+
+All of these components are then stored in a private repository on your Cloud 66 account so that you can roll back easily at any time.
+
+Snapshots act like a "time machine" for your code and configuration so you can redeploy your entire application as it was at any point in time or quickly revert your Kubernetes configurations if you need to.
+
+This also prevents issues when using the `latest` tag on your Docker images. If a `latest` tag is used in your service definition, a retagging will "freeze" that version at the time the snapshot is taken.
+
+**An infrastructure time machine**
+
+While Snapshots provide a way to "freeze" your code and other application components in time, they also make it possible to re-apply old code to new configuration. This is particularly useful in the following scenarios:
+
+* Rolling out old code a "playground" cluster for diagnostics.
+* Redeploying the same code with different versions of configuration for optimization purposes.
+* Rolling back configuration changes without redeploying the code.
+* Deploying the production version to a local cluster (like minikube) for debugging
+
+These are possible when code, configuration and all other parameters like environment variables are stored with Snapshots. 
+
+## What is Rendering?
+
+Rendering is the act of merging Stencils with a Snapshot and generating the resulting configuration files. Stencils of a Formation are rendered automatically when they are downloaded via the Cloud 66 UI or the Toolbelt CLI.
+
+<img src="/assets/skycap/rendering.png" width="200px"/>
+
+
+## What is a Base Template?
+
+Base Templates are a set of Stencils that can be used as the base for any new Formation. They contain the Stencils and their grouping. 
+
+For example you might choose to create a Base Template for your team so they can deploy the most commonly used components of your infrastructure to your Kubernetes cluster provider (for example AWS Fargate or Google GKE). 
+
+Base Templates can also group Stencils into arbitrary functional groups like Services and Configurations or Application, Databases, Storage, etc.
+
+Your Base Templates are stored in a git repository and are editing by your team or Cloud 66.
```

Comment:
`and are *editing* by your team` should be *edited*?

Reply:
Sorted, thanks!